normalizer/docker/parser.py

```diff
@@ -44,16 +46,27 @@ def __init__(
 
     def __str__(self):
         return self.content
-    
-    def insert_build_env(self, build_env):
+
+    def insert_build_env(self, build_env: Dict):
         build_env_str = ''
         for index, env in enumerate(build_env):
-            build_env_str += dedent(
-            f' --mount=type=secret,id={env} '
-        )
+            build_env_str += dedent(f' --mount=type=secret,id={env} ')
         for index, env in enumerate(build_env):
-            build_env_str += dedent(
-            f' export {env}=\"$(cat /run/secrets/{env})\" '
+            build_env_str += dedent(f' export {env}=\"$(cat /run/secrets/{env})\" ')
+        build_env_str += " && "
+
+        for index, line in enumerate(self._parser.lines):
+            strip_line = line.strip()
+            if RUN_VAR_RE.match(strip_line):
+                replace_line = line.replace('RUN', f'RUN {build_env_str}')
+                self._parser.content = self._parser.content.replace(line, replace_line)
+
+    def insert_build_env_path(self, build_env_path: pathlib.Path):
+        build_env_str = ''
+        build_env_path_str = str(build_env_path)
+
+        build_env_str += dedent(
+            f' --mount=type=secret,id={build_env_path.stem},dst={build_env_path} . {build_env_path_str} '
```

Comment:
The id and dst here are the same is this valid?
I think we can simply presume the secret name be `hubble_build_env` and load from `/run/secrets/hubble_build_env`

Reply:
> The id and dst here are the same is this valid?

mmm 'build_env_path.stem' means: `filename`;
```
build_env_path =  'src/build_env.sh'  
build_env_path.stem is `build_env`
```

Reply:
I see. But why we need this anyway, we could simply load from `/run/secrets/{id}` right?

Reply:
> I see. But why we need this anyway, we could simply load from `/run/secrets/{id}` right?

I will convert `--mount=type=secret,id={build_env_path.stem},dst={build_env_path} . {build_env_path_str} ` to`--mount=type=secret,id={build_env_file}. /run/secrets/{build_env_file}`